waspc/headless-test/README.md

```diff
@@ -0,0 +1,26 @@
+## Headless Test
```

Comment:
I think we could improve this file to be more informative! Some ideas:
1. Headless Test as a name is not very descriptive. If I asked you what is Headless test, how would you describe it to me? I would probably describe it as "It is when we take the whole wasp app and try to run it, you know clicking and stuff, end to end, see that it works as it should". Based on that, I would name it "app-e2e-test". We already have e2e-test/ dir btw, but that one is for cli (maybe it should be named `cli-e2e-test`). So this kind of fits in with that. I undrestand that "headless testing" is coming from "headless browser testing" but that is really an implementation detail of how these e2e tests are performed. I could use "headless" browser testing to test anything where I need to click in browser, so it is a technique, not precise explanation of what we are testing that way.
2. I would emphasize here in README that what we are doing here is taking an actual Wasp app, running it with wasp, and then running a suite of e2e tests on it, to confirm that basic functionality works as it should. And then list the signup, login and stuff you listed above.
3. You tell me to check out the test file -> why? What should I check out? If you want to say that is where most of the testing code is defined, you can say it that way.